Bump actions/checkout from 3 to 4 (#287)
rsksmart/devportal: tools/tokenbridge/dappguide.md

### Prerequisites

> This will require access to either Chrome or Chromium web browser, with one of the follwing wallet browser extension: 

Reviewer: ```suggestion
This will require the use of either Chrome or Chromium web browser, with one of the following wallet browser extensions: 
```

- [Nifty Wallet](https://www.poa.network/for-users/nifty-wallet) using a [custom network](/tutorials/resolve-nifty-issue/#add-rsk-as-custom-node) for RSK to keep the same addresses as Ethereum.

> - [Metamask](https://metamask.io/download.html) using a [custom network](/apps/wallets/metamask/) to add the RSK network.

Reviewer: ```suggestion
- [Metamask](https://metamask.io/download.html) using a [custom network](/develop/apps/wallets/metamask/) to add the RSK network.
```

> - [Nifty Wallet](https://www.poa.network/for-users/nifty-wallet) using a [custom network](https://developers.rsk.co/tutorials/resolve-nifty-issue/#add-rsk-as-custom-node) for RSK to keep the same addresses as Ethereum.
> - [Metamask](https://metamask.io/download.html) using a [custom network](https://developers.rsk.co/develop/apps/wallets/metamask/) to add the RSK network.

Reviewer: ```suggestion
- [Nifty Wallet](https://www.poa.network/for-users/nifty-wallet) using a [custom network](/tutorials/resolve-nifty-issue/#add-rsk-as-custom-node) for RSK to keep the same addresses as Ethereum.
- [Metamask](https://metamask.io/download.html) using a [custom network](/apps/wallets/metamask/) to add the RSK network.
```
